channel/consensus_channel/consensus_channel.go

```diff
@@ -428,3 +462,47 @@ func (v Vars) AsState(fp state.FixedPart) state.State {
 func (c *ConsensusChannel) Participants() []types.Address {
 	return c.fp.Participants
 }
+
+// CreateFromDirectFundingObjective accepts an objective and generates a new consensus channel from it.
+// It assumes that EVERY DirectFundingObjective is for a ledger channel.
+func CreateFromDirectFundingObjective(dfo directfund.Objective) (*ConsensusChannel, error) {
+	// The current assumption is that ANY direct funding objective is for a ledger channel
```

Comment:
You've stated this assumption twice. But I would argue the assumption applies to the code in the engine, not here? The engine could choose not to call this function for certain direct fund objectives.